web/src/engine/tsconfig.json

```diff
@@ -0,0 +1,37 @@
+{
+  "extends": "../tsconfig.base.json",
+
+	"compilerOptions": {
+		"outFile": "../../build/engine/obj/keymanweb.js"
+  },
+
+	"include": [
+    "./*.ts",
+    "./**/*.ts"
+  ],
+
+  "files": [
+		"kmwbase.ts",
+		"keymanweb.ts",
+		"kmwinit.ts",
+		"kmwapi.ts"
+  ],
+
+	"exclude": [
+		"./kmwnative.ts",
+		"./kmwembedded.ts",
+		"./kmwuibutton.ts",
+		"./kmwuifloat.ts",
+		"./kmwuitoggle.ts",
+		"./kmwuitoolbar.ts",
+	],
```

Comment:
There seems to be overlap in these definitions? Also are you planning to move the excluded files to another folder in a future PR?